### PR TITLE
refactor(schemas): remove refs of app schemas

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

File: src/schemas/ajvFormats.ts [hunk 1]
```diff
@@ -9,4 +9,7 @@ export default function plugin(ajv: Ajv) {
 
   // At least, 1 number, 1 lowercase and 1 uppercase letter. At least 8 characters.
   ajv.addFormat('strongPassword', /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}$/);
+
+  // At least, 1 number, 1 lowercase and 1 uppercase letter. At least 8 characters.
+  ajv.addFormat('strongPassword', /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}$/);
```
Comment (pyphilia) [inline src/schemas/ajvFormats.ts:14]: Duplicate?

**Review by pyphilia — CHANGES_REQUESTED** [review]

**Review by ReidyT — APPROVED** [review]
Really nice, LGTM! Good job 🫠 I just let you a small suggestion

File: src/services/item/plugins/app/appData/schemas.ts [hunk 2]
```diff
@@ -47,11 +46,11 @@ const create = {
     },
   },
   response: {
-    200: { $ref: 'https://graasp.org/apps/app-data/#/definitions/appData' },
+    200: appDataSchemaRef,
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:49]:
```suggestion
    [AtatusCodes.OK]: appDataSchemaRef,
```

File: src/services/item/plugins/app/appData/schemas.ts [hunk 3]
```diff
@@ -64,21 +63,21 @@ const updateOne = {
...
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:66]: Same here.

File: src/services/item/plugins/app/appData/schemas.ts [hunk 4]
```diff
@@ -64,21 +63,21 @@ const updateOne = {
...
 
-const deleteOne = {
+export const deleteOne = {
   params: Type.Object({
     itemId: customType.UUID(),
     id: customType.UUID(),
   }),
   response: {
-    200: { $ref: 'https://graasp.org/apps/app-data/#/definitions/appData' },
+    200: customType.UUID(),
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:76]: Also here

File: src/services/item/plugins/app/appData/schemas.ts [hunk 5]
```diff
@@ -90,11 +89,6 @@ const getForOne = {
     additionalProperties: false,
   },
   response: {
-    200: {
-      type: 'array',
-      items: { $ref: 'https://graasp.org/apps/app-data/#/definitions/appData' },
-    },
+    200: Type.Array(appDataSchemaRef),
```
Comment (spaenleh) [inline src/services/item/plugins/app/appData/schemas.ts:92]: Here too

**Review by spaenleh — APPROVED** [review]
Looks good !

**Review by pyphilia — APPROVED** [review]

**Pull request merged**